components/statuslight/stories/statuslight.stories.js

```diff
@@ -54,15 +61,71 @@ export default {
 		size: "m",
 		label: "Status",
 		variant: "info",
+		isDisabled: false,
 	},
 	parameters: {
 		packageJson: pkgJson,
 	},
 };
 
+/**
+ * Status lights should always include a label with text that clearly communicates the kind of status being shown. Color alone is not enough to communicate the status. Do not change the text color to match the dot.
+ * 
+ * When the text is too long for the horizontal space available, it wraps to form another line.
+ */
 export const Default = StatusLightGroup.bind({});
 Default.args = {};
 
+/**
+ * Status lights come in four different sizes: small, medium, large, and extra-large. The medium size is the default and most frequently used option. Use the other sizes sparingly; they should be used to create a hierarchy of importance within the page.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withBorder: false,
+	withHeading: false,
+	...args,
+}, context);
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disabledSnapshot: true },
```

Comment:
```suggestion
	chromatic: { disableSnapshot: true },
```

Oh I noticed these snapshots were being added! Can you update this and then I'll re-approve?

Reply:
gah! Good catch, thank you! https://github.com/adobe/spectrum-css/pull/3152/commits/d3cf45edddde5eb6be895f56fb8695899f9b2392

I left you a message in Slack about this, but what are your thoughts on me just dropping the disabled status light commit for now?